src/apps/AdminApp/AdminRoot.js

```diff
@@ -875,6 +881,71 @@ class AdminRoot extends Component {
       });
 
       let login = (<LoginPage user={user} checkPermission={this.checkPermission} />);
+		
+			const infoPopup =
+				<Popup trigger={<Button positive icon='info' onClick={this.getFeedInfo} />}
+					position='bottom left'
+					content={
+						<List as='ul'>
```

Comment:
Here you did not take list with latest change

Reply:
Fixed, please take a look.